dtos/device.go

```diff
@@ -45,8 +47,10 @@ type UpdateDevice struct {
 // ToDeviceModel transforms the Device DTO to the Device Model
 func ToDeviceModel(dto Device) models.Device {
 	var d models.Device
+	d.DBTimestamp = models.DBTimestamp(dto.DBTimestamp)
```

Comment:
Noticed this was missing when adding the new unit test.

Reply:
... and it occurs to me that we probably don't want DBTimestamp to propagate here... will update.